target/linux/ipq40xx/config-5.4

```diff
@@ -536,3 +536,4 @@ CONFIG_ZLIB_DEFLATE=y
 CONFIG_ZLIB_INFLATE=y
 CONFIG_ZSTD_COMPRESS=y
 CONFIG_ZSTD_DECOMPRESS=y
+CONFIG_IPQ40XX_PORT_ISOLATION=n
```

Comment:
I think that is unnecsessary and prevents setting KCONFIG:= in image.mk... just testing...